src/integrationTest/java/org/opensearch/test/framework/TestSecurityConfig.java

```diff
@@ -58,6 +59,9 @@
 import org.opensearch.core.xcontent.XContentBuilder;
 import org.opensearch.security.securityconf.impl.CType;
 import org.opensearch.test.framework.cluster.OpenSearchClientProvider.UserCredentialsHolder;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import org.opensearch.core.common.io.stream.Writeable;
```

Comment:
I don't think you need any of these other references, can they be removed?

Reply:
Removed this. I thought this was necessary in an earlier iteration, but its not needed.